cli/muxer.c

```diff
@@ -277,6 +284,9 @@ static void display_help( void )
              "    disable                   Disable this track\n"
              "    dv-profile=<arg>          Specify Dolby Vision profile\n"
              "    ambisonic-order=<integer> Specify ambisonic order for a spatial audio track\n"
+             "    stereo-mode=<integer>     Specify stereoscopic 3d mode on the video track\n"
```

Comment:
Minor consistency nits

```suggestion
             "    stereo-mode=<integer>     Specify stereoscopic 3D mode on the video track\n"
```

Reply:
ok